managed/services/agents/rds.go

```diff
@@ -22,11 +22,12 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
 	"gopkg.in/yaml.v3"
-
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
 	"github.com/percona/pmm/managed/models"
 	"github.com/percona/pmm/version"
+	"os"
+	"fmt"
```

Comment:
please move these import to the first block

Reply:
Like now? May be i something haven't understood?